tasks/sign-index-image/tests/test-sign-index-image.yaml

```diff
@@ -37,9 +37,9 @@ spec:
         - name: requester
           value: testuser
         - name: referenceImage
-          value: quay.io/testrepo/testimage:tag
+          value: quay.io/redhat-pending/testimage:tag
```

Comment:
The test failed:
```
[run-task : sign-index-image] ++ translate-delivery-repo quay.io/redhat-pending/testimage:tag
[run-task : sign-index-image] + reference_image='Malformatted repo to translate. Expected format is quay.io/redhat-[prod,pending]/product----repo'
```
So it will have to be something like `quay.io/redhat-pending/testproduct----testimage:tag`

Reply:
Oh I see you're trying to solve this with https://github.com/redhat-appstudio/release-service-utils/pull/119 , right? Not sure what the right thing to do here is. But yeah, probably no need to fail.